Integration test for sample aggragator

nik9000 opened the pull request:
Adds and integration test for the sample aggregator. It's a fairly
simple aggregator - you can only chose how many documents it samples.

elasticmachine commented:
Pinging @elastic/es-analytics-geo (Team:Analytics)

not-napoleon reviewed (approved):
Thanks for adding this, LGTM

nik9000 committed:
Integration test for sample aggragator

Adds and integration test for the sample aggregator. It's a fairly
simple aggregator - you can only chose how many documents it samples.
